src/DXRModules/DeusEx/Classes/DXREvents.uc

```diff
@@ -2521,16 +2523,16 @@ static simulated function string GetBingoGoalHelpText(string event,int mission,
     switch(event){
         case "Free Space":
             return "Don't worry about it!  This one's free!";
-        case "TerroristCommander_Dead":
-            return "Kill Leo Gold, the terrorist commander on Liberty Island";
+        case "TerroristCommander_PlayerDead":
+            return "Kill Leo Gold, the terrorist commander on Liberty Island.  You must kill him yourself.";
```

Comment:
all of these need the old names retained to give help for people with old save files (they don't need to be perfect, it can be the new description so the players start getting used to doing the goals properly)

```
        case "TerroristCommander_Dead":
        case "TerroristCommander_PlayerDead":
            return "Kill Leo Gold, the terrorist commander on Liberty Island.  You must kill him yourself.";
```